src/ClientBase.php

```diff
@@ -412,4 +428,35 @@ protected function verifyDates($from, $to, $format = 'YmdHs')
             throw new \InvalidArgumentException($message);
         }
     }
+
+    /**
+     * Helper function to extract the option codes.
+     *
+     * @param array $options
+     *   The options array.
+     *
+     * @return array
+     *   The list of options with the option-code.
+     */
+    protected function parseOptionCodes(array $options)
+    {
+        $valid_options = [];
+        foreach ($options['option_codes'] as $optionCode) {
+            if (!in_array(strtoupper($optionCode), self::getOptionCodes())) {
```

Comment:
**CHANGE REQUEST**

The exception we had added here has been lost somewhere while moving this function from one file to another.